build.sbt

```diff
@@ -126,6 +126,40 @@ lazy val checkJavaVersion = taskKey[Unit]("Verifies the current Java version is
 
 lazy val makeProductPage = taskKey[Unit]("Crete product page")
 
+lazy val productPage = project
+  .in(file("product-page"))
+  .enablePlugins(GhpagesPlugin)
+  .settings(shared, compilation)
+  .settings(
+    name                              := "product-page",
+    moduleName                        := "product-page",
+    // gh pages settings
+    git.remoteRepo                    := "git@github.com:BlueBrain/nexus.git",
+    ghpagesNoJekyll                   := true,
+    ghpagesBranch                     := "gh-pages",
+    makeProductPage                   := {
+      import scala.sys.process._
+      import java.nio.file.Files
+      val log     = streams.value.log
+      if (!Files.exists(siteSourceDirectory.value.toPath)) Files.createDirectory(siteSourceDirectory.value.toPath)
+      val install = Process(Seq("make", "install"), baseDirectory.value / "src" / "product-page-src")
+      val build   = Process(Seq("make", "build"), baseDirectory.value / "src" / "product-page-src")
+      if ((install #&& build !) == 0) {
+        log.success("Product page built.")
+      } else {
+        log.error("Product page built failed.")
+        throw new RuntimeException
+      }
+    },
+    makeSite                          := makeSite.dependsOn(makeProductPage).value,
```

Comment:
I'm not sure this is sufficient; we may need to add site mappings to all the files generated in the site dir.